.github/workflows/unit-test.yml

```diff
@@ -16,11 +17,12 @@ jobs:
         run: sudo service mysql stop # Shutdown the Default MySQL, "sudo" is necessary, please not remove it
 
       - name: Set up MySQL
-        uses: mirromutth/mysql-action@v1.1
+        uses: shogo82148/actions-setup-mysql@v1
```

Comment:
I'll be honest. It makes me a little nervous importing unofficial gitactions.

Is it possible to use docker to run mysql?

Reply:
I tried docker and it takes around 24 minutes to do unit tests